Home page template

nienn opened the pull request:
Again, just moving code from mockups to templates.

* Merge after #26

agjohnson commented:
It seems this is blocked on the copy conversation in #23

agjohnson commented:
Copy conversation is merged, this can continue.

agjohnson commented on readthedocs-theme/templates/readthedocs/homepage.html:
```diff
@@ -3,14 +3,119 @@
+                  <img src="/theme/img/infographic-01.png"/> 
+                </div>
+                <p>
+                  Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what maters: your project.
```
Hrm. I noted in the previous PR that this should be Markdown and reStructuredText, not `.md or .rst`. Seems we missed some of the review feedback in the previous PR. Is this not yet ready for review?

agjohnson commented on readthedocs-theme/templates/readthedocs/homepage.html:
```diff
@@ -3,14 +3,119 @@
+                  <img src="/theme/img/infographic-03.png"/>
+                </div>
+                <p>
+                  Setup your account with Read the Docs, add a .config file to tell us your preferences and we'll do the rest!
```
Yeah, here too, this is missing review contributions from the last PR.

agjohnson reviewed (commented):
It's not clear if this was ready for review, but I'm noticing missing copy changes from the previous PR.

nienn commented:
Ups, I just noticed that Juan's PR [#23](https://github.com/readthedocs/site-community/pull/23/files?diff=unified&w=0#) was done over the `mockup/` files and, of course, that would not update the `templates/` when I rebased. It's now rebased and I ported the changes to the templates manually. Should be ready for review now.

Same in https://github.com/readthedocs/site-community/pull/28.

agjohnson reviewed (approved):
Looks good to merge! We'll do another round of changes after the existing pages are merged in.
